packages/zoe/test/unitTests/test-zoe.js

```diff
@@ -415,7 +418,7 @@ test(`zoe.getConfiguration`, async t => {
     },
     meteringConfig: {
       incrementBy: 25000000n,
-      initial: 50000000n,
+      initial: 75000000n,
```

Comment:
This test detects any change to the default metering config, which isn’t terribly valuable. Could this instead import the defaults and inject them back here?